plugin/src/main/java/net/dzikoysk/funnyguilds/feature/hooks/mvdwplaceholderapi/MVdWPlaceholderAPIHook.java

```diff
@@ -68,38 +69,38 @@ public HookInitResult init() {
 
         // Guild TOP, positions 1-100
         for (int i = 1; i <= 100; i++) {
-            final int position = i;
+            int position = i;
 
-            guildTopIds.forEach(id ->
-                    PlaceholderAPI.registerPlaceholder(plugin, "funnyguilds_gtop-" + id + "-" + position, event -> {
+            PandaStream.of(guildTopIds).forEach(id ->
+                    PlaceholderAPI.registerPlaceholder(this.plugin, "funnyguilds_gtop-" + id + "-" + position, event -> {
                         User user = userManager.findByPlayer(event.getPlayer()).orNull();
-                        return rankPlaceholdersService.formatTop("{GTOP-" + id.toUpperCase() + "-" + position + "}", user);
+                        return rankPlaceholdersService.formatTop("{GTOP-" + id.toUpperCase(Locale.ROOT) + "-" + position + "}", user);
                     }));
 
             if (pluginConfiguration.top.enableLegacyPlaceholders) {
-                PlaceholderAPI.registerPlaceholder(plugin, "funnyguilds_gtop-" + position, event -> {
+                PlaceholderAPI.registerPlaceholder(this.plugin, "funnyguilds_gtop-" + position, event -> {
                     User user = userManager.findByPlayer(event.getPlayer()).orNull();
                     return rankPlaceholdersService.formatRank("{GTOP-" + position + "}", user);
                 });
             }
         }
 
-        userTopIds.forEach(id ->
-                PlaceholderAPI.registerPlaceholder(plugin, "funnyguilds_position-" + id, event -> {
+        PandaStream.of(userTopIds).forEach(id ->
+                PlaceholderAPI.registerPlaceholder(this.plugin, "funnyguilds_position-" + id, event -> {
                     User user = userManager.findByPlayer(event.getPlayer()).orNull();
-                    return rankPlaceholdersService.formatTopPosition("{POSITION-" + id.toUpperCase() + "}", user);
+                    return rankPlaceholdersService.formatTopPosition("{POSITION-" + id.toUpperCase(Locale.ROOT) + "}", user);
                 }));
 
-        guildTopIds.forEach(id ->
-                PlaceholderAPI.registerPlaceholder(plugin, "funnyguilds_g-position-" + id, event -> {
+        PandaStream.of(guildTopIds).forEach(id ->
+                PlaceholderAPI.registerPlaceholder(this.plugin, "funnyguilds_g-position-" + id, event -> {
                     User user = userManager.findByPlayer(event.getPlayer()).orNull();
-                    return rankPlaceholdersService.formatTopPosition("{G-POSITION-" + id.toUpperCase() + "}", user);
+                    return rankPlaceholdersService.formatTopPosition("{G-POSITION-" + id.toUpperCase(Locale.ROOT) + "}", user);
                 }));
 
         return HookInitResult.SUCCESS;
     }
 
-    public String replacePlaceholders(Player user, String base) {
+    public static String replacePlaceholders(Player user, String base) {
```

Comment:
Pakowanie tego w static jest raczej nie wskazane.
Jeśli instancja nie zostanie stworzona to nie można użyć replacePlaceholders
w tym przypadku można jej używać dowolnie.